checkall.sh

```diff
@@ -37,7 +37,7 @@ echo "Performing checkup:"
 
 # cppcheck
 cppcheck --version
-cppcheck -iclang-format-report.txt -iclang-tidy-report.txt --enable=all --library=qt.cfg --std=c++17 --language=c++ --output-file=cppcheck-report.txt ./files/*.{cpp,h,c,hpp}
+cppcheck -iclang-format-report.txt -iclang-tidy-report.txt --enable=all --library=qt.cfg --std=c++17 --language=c++ --output-file=cppcheck-report.txt ./files/*.{cpp,c}
```

Comment:
hpp, in my experience are header file that do contains codes, are we sure we don't want to parse them?

Reply:
you're right, but it seems we don't write hpp files in our projects anyway, do we? so maybe keep them out? (unless it doesn't impact the search speed by any means).

Either is fine anyway, the key thing is to remove the `h`